perun-core/src/test/java/cz/metacentrum/perun/taskslib/dao/ServicesManagerBlImplTest.java

```diff
@@ -39,10 +41,9 @@
 @ContextConfiguration(locations = { "classpath:perun-base.xml", "classpath:perun-core.xml" })
 })
 @Transactional(transactionManager = "springTransactionManager")
-public class ServiceDenialDaoTest {
+public class ServicesManagerBlImplTest {
```

Comment:
It would make more sense, if this test class would be moved to the core package, since you are testing methods of core services manager now.
Also I would prefer these tests to be part of Entry layer services manager test class, but it is probably to much work now. Just move the class to proper package, add TODO message about moving its content to normal services managers tests and remove all unnecessary getters/setters (which are not related to ``@autowired`` properties).